popout.js

```diff
@@ -1,42 +1,44 @@
 class PopoutModule {
 	static onRenderJournalSheet(obj, html, data) {
-		let element = html.find(".window-header .window-title")
-		PopoutModule.addPopout(element, `game.journal.get("${obj.entity.id}").sheet`);
+		let element = html.find(".window-header .window-title");
+		let sheetID = obj.entity.id;
+		PopoutModule.addPopout(element, `game.journal.get("${obj.entity.id}").sheet`, sheetID);

```

Comment:
The object itself should be passed to `AddPopout`, rather than its id.

Reply:
If you pass the object to AddPopout, isn't that going to add the entire object to the little button instead of just a reference to the ID?

Reply:
It doesn't add it to the button, it just keeps a reference to the object within the scope of the function callback to be called. It won't use more RAM or anything either since the object still lives on within foundry itself and once the sheet is closed, the button and its click handler is garbage collected along with the function and its scope.